packages/data/src/onboarding/resolvers.js

```diff
@@ -60,14 +68,77 @@ export function* getTasksStatus() {
 	}
 }
 
+function getQuery() {
+	const searchString = window.location && window.location.search;
+	if ( ! searchString ) {
+		return {};
+	}
+
+	const search = searchString.substring( 1 );
+	return parse( search );
+}
+
+/**
+ * This function will be depreciated in favour of registering tasks on the back-end.
+ *
+ * @param {Array} taskLists array of task lists.
+ */
+function* mergeWithFilteredTasks( taskLists ) {
```

Comment:
The naming on this threw me for a second, but I'm not really able to come up with much better.  `mergeWithTasksFromFilter`?  Not sure if that's an improvement or not.